tests/support/plugins/file_server.py

```diff
@@ -6,7 +6,7 @@
 #-----------------------------------------------------------------------------
 ''' Define a simple web server for testing purpose.
 
-Used for serves the testing html pages that are needed by the webdriver unit
+Used for servers testing the html pages that are needed by the webdriver unit
```

Comment:
```suggestion
Serve html pages that are needed by the webdriver unit tests.
```

I think this was the intended meaning
(and delete the "tests." on the following line